atlas/packages/app/src/shared/components/Button/Button.style.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react'
+import { css } from '@emotion/core'
+import styled from '@emotion/styled'
+import { BlockIcon } from '@/shared/icons'
+import { typography, colors, spacing } from '../../theme'
+
+export type ButtonStyleProps = Partial<{
+  variant: 'primary' | 'secondary' | 'tertiary'
+  full: boolean
+  size: 'regular' | 'small' | 'smaller'
+  children: React.ReactNode
+  onClick: (e: React.MouseEvent<HTMLButtonElement>) => void
+  disabled: boolean
+  hoverable: boolean
+  clickable: boolean
+  pressedDown: boolean
+}>
+
+const colorsFromProps = ({ variant, hoverable = false, pressedDown }: ButtonStyleProps) => {
```

Comment:
There's a bug that will make secondary button look like a primary one after clicking:


![Screen-Recording-2020-07-30-at-5 59 28-PM](https://user-images.githubusercontent.com/12646744/88945910-dccf8d00-d28e-11ea-8388-4934f0ecea4d.gif)

Reply:
Not a bug, that's the pressed state for a secondary button, as seen here: https://www.dropbox.com/s/mx2wminhpwxx7by/Screen%20Recording%202020-06-04%20at%2011.14.42.mov?dl=0

Reply:
I see, that looks like a "toggled" state I've talked about before though. I presumed it's a bug because other buttons on the same page don't have this behavior. So here I'd say the same thing about button being "toggleable" as here: https://github.com/Joystream/joystream/pull/977#discussion_r463435760

Per the "Components" screen in Figma, a regular secondary button should just have a different color of the text, not a blue background.

Reply:
Other buttons on that page don't have this behaviour because they don't have an `onClick` prop, so `setPressedDown` is never called. 

Regarding the adherence to the design, if it is toggled, it should look like this: https://www.dropbox.com/s/mx2wminhpwxx7by/Screen%20Recording%202020-06-04%20at%2011.14.42.mov?dl=0